.github/workflows/ci.yml

```diff
@@ -29,7 +29,9 @@ jobs:
         with:
           components: rustfmt, clippy
       - name: ci-formatting
```

Comment:
This could be named "check-format", given that there is no ci-formatting rule in the Makefile.